TAO/TAO_IDL/docs/annotations.md

```diff
@@ -87,15 +87,20 @@ of what can be annotated and how, annotations for specific IDL elements have to
 be added on a as-needed basis.
 
 You can put annotations before the following things in IDL and a backend using
-TAO IDL's front-end library can read them:
+TAO IDLs front-end library can read them:
```

Comment:
Why did this change?

Reply:
I think I accidentally deleted it in vim and didn't notice. It's easy to do stuff like this when you think your in insert mode but you're actually in command mode.